src/strategies/spidStrategy.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from "../utils/idpLoader";
 import { log } from "../utils/logger";
 
-const IDP_IDS: { [key: string]: string | undefined } = {
+export const IDP_IDS: { [key: string]: string | undefined } = {
```

Comment:
why `| undefined` ? cannot we just use `Record<string, string> `here ?

Reply:
The type `Record<string, string>` can't map the case when a specific key is missing in the object:
```typescript
const IDP_IDS: Record<string, string> = {
"test": "value"
};

const undefinedValue = IDP_IDS["missing_key"]; // The type is string but the value is undefined
```